src/ddscxx/src/org/eclipse/cyclonedds/core/policy/PolicyDelegate.cpp

```diff
@@ -876,6 +876,87 @@ void OwnershipDelegate::set_c_policy(dds_qos_t* qos) const
 }
 
 
+//==============================================================================
+
+DataRepresentationDelegate::DataRepresentationDelegate(const DataRepresentationDelegate& other)
+    : kinds_(other.kinds_)
+{
+}
+
+DataRepresentationDelegate::DataRepresentationDelegate(const std::vector<dds::core::policy::DataRepresentationKind::Type>& kinds)
```

Comment:
Please replace the anonymous vector with its proper typedef here. And also replace the name "kinds" with "value".

Reply:
Will do.